amun/base/argo-workflows/inspection-build-with-cpu.yaml

```diff
@@ -234,13 +234,16 @@ spec:
           results="/mnt/build/"
           mkdir -p "$results"
           kubectl logs "{{inputs.parameters.inspection-id}}-1-build" > "${results}/log"
-          cat <<EOFEOF  | jq . > "${results}/specification"
+          cat << EOF  | jq . > "${results}/specification"
           {{inputs.parameters.specification}}
-          EOFEOF
-          cat <<EOFEOF > "${results}/Dockerfile"
-          {{inputs.parameters.dockerfile}}
-          EOFEOF
+          EOF
```

Comment:
is this EOF intended?

Reply:
I think so, it's corresponding to the `cat << EOF`.